src/trans-word-level/property.cpp

```diff
@@ -256,6 +256,17 @@ static obligationst property_obligations_rec(
 
     obligationst obligations;
 
+    // Traces with any φ state from "current" onwards satisfy Fφ
+    exprt::operandst phi_disjuncts;
```

Comment:
We should make a habit of using `.reserve` when populating the container via `push_back` a known quantity of times. It'll take an extra type conversion here to go from `mp_integer` to `std::size_t`, but we should avoid the potentially quadratic cost of re-allocating and copying.

Reply:
Done